packages/wallet-sdk/src/mpc-contract.ts

```
@@ -0,0 +1,68 @@
+            }
+        }
+
+        const argsJson = JSON.stringify({
+            request: {
+                payload_v2,
+                domain_id: domainId,
+                path
+            }
+        });
+
+        console.log(`Domain ID ${argsJson}`)
```

Biome (2.4.4)

[error] Don't use console.

(lint/suspicious/noConsole)